src/sage/matrix/matrix_polynomial_dense.pyx

```diff
@@ -4079,3 +4084,564 @@ cdef class Matrix_polynomial_dense(Matrix_generic_dense):
                 if P[j, j].degree() + shifts[j] <= degree_bound:
                     column_indices.append(j)
             return P[:,column_indices]
+
+    def _basis_completion_via_reversed_approx(self):
+        r"""
+        Return a Smith form-preserving nonsingular completion of a row basis of
+        this matrix. For a more detailed description, see
+        :meth:`basis_completion`, in the row-wise case.
+
+        EXAMPLES::
```

Comment:
```suggestion
        EXAMPLES:
```

This should be single colon since there are no tests *immediately* after this indicator. The double colon below (... unimodular matrix::) is correct.

Reply:
thanks!